src/validation/fields/helper/date.validation.helper.ts

```diff
@@ -43,6 +44,23 @@ export const conditionalDateValidations = (dateContextWithCondition: dateContext
   ];
 };
 
+export const conditionalHistoricalBODateValidations = (trustDateContext: dateContextWithCondition) => [
+  setValidateCeasedDateFlag(trustDateContext),
+  ...conditionalDateValidations(trustDateContext),
+];
+
+export const setValidateCeasedDateFlag = (trustDateContext: dateContextWithCondition) => {
+  return (req: Request, resp: Response, next: NextFunction) => {
+    const isUnableToProvideAllTrust: boolean = isUnableToObtainAllTrustInfo(req);
```

Comment:
the const isUnableToProvideAllTrust should have "Info" on the end

Reply:
Done